.github/workflows/publish-release.yml

```diff
@@ -33,6 +33,11 @@ jobs:
               body: releaseNotes.body,
             })
 
+            console.debug('Release notes:', JSON.stringify(releaseNotes.body))
```

Comment:
Yep, because the HTML comment doesn't get rendered in the summary (which is fine), but I want to easily see it if necessary:

<img width="1153" alt="image" src="https://github.com/DataDog/datadog-ci/assets/9317502/0c0b2577-13b4-4f77-92e6-a0031fe11d0e">